### ISSUE TITLE
Remember the Milk plugin authentication

### ISSUE THREAD

**Issue by StuartRob** [quicksilver/Quicksilver#2115]
I think the API problem with the Remember the Milk pPlugin has reappeared. The plugin fails to authenticate and get the list of tasks and lists. When I try to re-authenticate I get the webpage that says 'No callback URL specified for this API key. Your frob value' and then a long string of numbers and letters. There's a closed issue from February which is the same problem I think.

**Comment by pjrobertson** [comment]
I have just tried this, and it works fine for me. I installed the plugin, went to the prefs and clicked 'start' and the webpage opened asking me to login.

**Comment by StuartRob** [comment]
I installed the plugin, went to the prefs and clicked 'start' and the webpage opened asking me to login. I logged in and saw a page for authorising the plugin. I authorise the plugin but when I go back to the prefs and click 'finish' I get the message 'There was an error ((null)). Make sure that you have given this plugin permission to access Remember the Milk'. This is the same error that I received previously.

![screen shot 2015-10-18 at 15 04 43](https://cloud.githubusercontent.com/assets/1903755/10564211/7c1a6678-75ab-11e5-8306-c50abf028fd2.png)
